src/CoreBundle/Form/Type/BooleanType.php

```diff
@@ -19,16 +19,13 @@
     );
 }
 
-class_alias(
-    \Sonata\Form\Type\BooleanType::class,
-    __NAMESPACE__.'\BooleanType'
-);
-
-if (false) {
-    /**
-     * @deprecated Since version 3.x, to be removed in 4.0.
-     */
-    class BooleanType extends \Sonata\Form\Type\BooleanType
+/**
+ * @deprecated Since version 3.x, to be removed in 4.0.
+ */
+class BooleanType extends \Sonata\Form\Type\BooleanType
+{
+    public function getBlockPrefix()
     {
+        return 'sonata_type_boolean_legacy';
```

Comment:
The block prefix changes, meaning every override associated with this prefix will break, doesn't it?